tests/aws/services/sqs/test_sqs.py

```diff
@@ -251,34 +262,33 @@ def test_send_receive_message_multiple_queues(self, sqs_create_queue, aws_client
         aws_client.sqs.send_message(QueueUrl=queue0, MessageBody="message")
 
         result = aws_client.sqs.receive_message(QueueUrl=queue1)
-        assert "Messages" in result
-        assert result["Messages"] == []
+        assert "Messages" not in result or result["Messages"] == []
```

Comment:
For empty lists, AWS often omits the field entirely, independent of the protocol. I noticed this behavior recently while parity testing different SDK versions (which use different protocols) in the Lambda multiruntime tests.

I moved my Notion testing page to the SQS service to demonstrate the behavior for the `listQueues` operation (see [here](https://www.notion.so/localstack/SQS-SDK-parity-testing-164bfb9dbdd541928fd2b2abc275bac7?pvs=4)). It looks like we have an SQS parity issue when listing empty queues (unrelated to this PR).

Reply:
Thanks for the comment! Since this is tested in dozens of the snapshot tests (specifically for each individual protocol due to the new fixture being used in the tests), I wanted to make these assertions independent of the protocol. Also because the next switch is around the corner (I'm quite sure the `botocore` team will try another run on the switch somewhat soon).
Also, it seems like they are currently actually changing the behavior server-side (the `json` service is _not_ returning empty lists anymore). But a gain, I think this is something which should belong in the snapshot verification. Here we actually semantically want to check if there are no messages left (and not explicitly want to test the protocol).

I'll close the other comments concerning this, but I'm happy to discuss this. If we want to change this, then obviously I'll apply it everywhere.